game/src/core/System.java

```diff
@@ -61,11 +61,12 @@ public abstract class System {
      * @param filterRules Needed Component-Classes. Entities need the components to be processed by
      *     this system.
      */
+    @SafeVarargs
     public System(Class<? extends Component>... filterRules) {
         if (filterRules != null) this.filterRules = Set.of(filterRules);
         else this.filterRules = new HashSet<>();
         run = true;
-        LOGGER.info("A new " + this.getClass().getName() + " was created");
+        LOGGER.info(String.format("A new %s was created", getClass().getName()));
```

Comment:
mmmh entweder immer Format verwenden oder immer concat das mischen kommt mir gerade nichtz logisch vor

Reply:
you are right